graphics/Quarter/Portfile

```
@@ -1,33 +1,78 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           qt4 1.0
 
+PortGroup           active_variants 1.1
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+# tag_prefix will be changed to v starting with version 1.1.1
+github.setup        coin3d quarter 1.1.0 Quarter-
+github.tarball_from releases
 name                Quarter
-version             1.0.0
-revision            5
+
 categories          graphics devel
-license             GPL-2
 platforms           darwin
+revision            0
 maintainers         nomaintainer
-
+license             BSD
 description         integrates Coin and Qt
 
 long_description    Quarter is a light-weight glue library that provides \
                     seamless integration between Kongsberg SIM's Coin \
                     high-level 3D visualization library and Nokia's Qt 2D \
                     user interface library.
 
-homepage            http://www.coin3d.org/lib/quarter/
-master_sites        http://ftp.coin3d.org/coin/src/all/
+homepage            https://coin3d.github.io
 
-checksums           md5     73d7c9e1917436e29b128781c997185f \
-                    sha1    f141490008c3139cb8f01c71148e2dd8802a97d0 \
-                    rmd160  d954cac8d35e83ffb565c28fd3e28864b8dfa023
+distname            quarter-${version}-src
+distfiles           quarter-${version}-src${extract.suffix}
+
+checksums           rmd160  a81b269ac0cb7c0c9e8b27659744d854b81ccf4d \
+                    sha256  d055d6c2af199f0cad592d24649f1d7acf842c88741ff74a6f7ff37675a9dd01 \
+                    size    724762
 
 depends_lib-append  port:Coin
 
-configure.ldflags-append    -L${qt_libs_dir}
-configure.cppflags-append   -I${qt_includes_dir}
-configure.args-append       --with-qt-designer-plugin-path=${qt_plugins_dir}/designer \
-                            --without-framework
+patchfiles          0001-cpack.d-is-now-optional.patch \
+                    0002-remove-QT_VERSION_CHECK.patch
+
+configure.args-append   -DCMAKE_VERBOSE_MAKEFILE=ON \
+                        -DQUARTER_BUILD_PLUGIN=ON \
```

This line can be removed because the cmake 1.1 portgroup already sets it for you in `configure.pre_args`.
```suggestion
configure.args-append   -DQUARTER_BUILD_PLUGIN=ON \
```